csv_datasets/random_graphs/small_graph/nodes_empty.txt

```diff
@@ -0,0 +1,2 @@
+ID:ID(PERSON),NAME:STRING
```

Comment:
this looks the same as nodes_empty_nodata.txt can we just use one (two times)?

Reply:
updated the data in the file.